baybe/utils/scaling.py

```diff
@@ -0,0 +1,37 @@
+"""Scaling utilities."""
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import TYPE_CHECKING, Literal, TypeAlias
+
+if TYPE_CHECKING:
+    from sklearn.base import BaseEstimator, TransformerMixin
+
+    Scaler: TypeAlias = BaseEstimator | TransformerMixin
+
+
+class ScalingMethod(Enum):
+    """Available scaling methods."""
+
+    IDENTITY = "IDENTITY"
+    """Identity transformation (no scaling applied)."""
+
+    MINMAX = "MINMAX"
+    """Min-max scaling, mapping the observed value range to [0, 1]."""
+
+    MAXABS = "MAXABS"
+    """Max-abs scaling, normalizing by the largest observed absolute."""
+
+
+def make_scaler(method: ScalingMethod, /) -> Scaler | Literal["passthrough"]:
+    """Create a scaler object based on the specified method."""
+    from sklearn.preprocessing import MaxAbsScaler, MinMaxScaler
```

Comment:
why lazy

Reply:
For context: my goal is not to make sklearn optional, but at some point we can still make it entirery lazy-loaded, since this is now causing by far the biggest loading time. But it's one of my "not in the mood for serious coding" items which will require a separate short PR anyway, so I'm happy to change it here if it bothers you

Reply:
ok I see thats ok then, 
ngl the way that its now inconsistently lazy/eager in various code parts bothers me `slightly`, id prefer to make these changes then in a dedicated PR and not already half-way here, but if its unnecessary load to remove it again then forget about it and resolve

Reply:
already changed it back now worries 👍🏼